BankingApplication/src/main/java/com/ark/bankingapplication/Controller.java

```diff
@@ -224,4 +224,19 @@ public boolean logout(String sessionKey) {
             return false;
         }
     }
+
+    public void sessionExpired() {
```

Comment:
Noem dit sessionTerminated.... dat is waar je op abonneert. Makkelijker terug te vinden en daarbij zegt het wat het doet. Expired hoeft niet het geval te zijn namelijk.